File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        int n1=s.length();
        int n2=t.length();
        if(n1!=n2)
        return false;
        int []sc=new int[26];
        int []tc=new int[26];
        for(int i=0;i<s.length();i++)
        {
            sc[s.charAt(i)-'a']++;
            tc[t.charAt(i)-'a']++;
        }
         for(int i=0;i<26;i++)
        {
            if(sc[i]!=tc[i])
            return false;
        }
        return true;
        
    }
}

File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        ar1=[0]*26

        if len(s)!=len(t):
            return False
        for i in range (len(s)):
            ar1[ord(s[i])-ord('a')]+=1
            ar1[ord(t[i])-ord('a')]-=1
        for i in range(26):
            if ar1[i]!=0:
                return False
        return True

        